homeassistant/components/hive/binary_sensor.py

```diff
@@ -91,3 +128,17 @@ async def async_update(self) -> None:
             self._attr_available = self.device["deviceData"].get("online")
         else:
             self._attr_available = True
+
+class HiveSensorEntity(HiveEntity, SensorEntity):
```

Comment:
It should still inherit BinarySensorEntity

Reply:
Completed